### DISCUSSION TITLE
ESP32 new version hardware devices ESP32S3, EPS32C3, ESP32S2

### DISCUSSION THREAD

**Discussion by deepwater-code** [cesanta/mongoose#2178] [Ask questions, get answers] [answered]
Hello,

We are developing some new IoT hardware and would like to use the ESP32S3 plus the ESP32C3. It does not appear that these are currently supported, will these devices be supported in the near future?

Thank you for your help

**Comment by scaprile** [comment]
The fact that there is no ready-made example doesn't necessarily mean it won't work, just that we didn't play with that one, yet.
If it is supported by the Espressif IDF you should be able to make it work. I suggest you try with a cheap breadboard and let us know if you face any issues.

**Comment by cpq** [comment, marked as answer]
@deepwater-code they are all supported. They are just not present in the examples.
